conjure/conjure.go

```diff
@@ -129,3 +134,7 @@ func newGoFile(filePath string, file *jen.File) *OutputFile {
 		file:    file,
 	}
 }
+
+func packageSuffixRequiresAlias(importPath string) bool {
+	return regexp.MustCompile(`/v[0-9]+$`).MatchString(importPath)
```

Comment:
This looks great, thanks! The implementation the Go team uses for this is at https://github.com/palantir/go-ptimports/blob/93cb0022ccc2ca0a21b3ac3396b85349567cc40a/vendor/golang.org/x/tools/internal/imports/fix.go#L1145-L1153, but I think this regexp matching is more readable and fine.